docusaurus/docusaurus.config.js

```diff
@@ -503,6 +503,37 @@ const config = {
           //   label: 'Blog',
           //   position: 'right'
           // },
+          {
+            type: 'dropdown',
+            label: 'Docs',
+            position: 'right',
+            items: [
+              {
+                label: 'Blog',
+                href: 'https://openziti.io',
+              },
+              {
+                label: 'YouTube',
+                href: 'https://www.youtube.com/openziti',
+              },
+              {
+                label: 'Forum',
+                href: 'https://openziti.discourse.group',
+              },
+              {
+                label: 'Twitter',
+                href: 'https://twitter.com/openziti',
+              },
+              {
+                label: 'Ziggy\'s Twitter',
```

Comment:
I'm torn on ziggy. maybe just put him last - also can there be dividors added for separation of stuff like that? we could also add the test kitchen with chef ziggy closeup. something close to this (which is pretty close to what is there already)
![image](https://user-images.githubusercontent.com/46322585/206755706-b77a3e6a-6e18-4c24-9de8-bb6b3271eb55.png)

Reply:
for css, merge my other branch into this one and it'll be clear how to add others. i'm on the fence over the NetFoundry linked in link.

Reply:
There's already a CloudZiti link. That seems sufficient.